### PR TITLE
Visually hidden div for accessibility 

### PR DESCRIPTION
**Pull request by chasulin**
- Extracts `VisuallyHidden` div into something more generic + reusable for accessibility
- Uses uuid to give each tooltip a unique ID for `aria-describedby` attribute

### CODE REVIEW

File: src/components/InfoTooltip/DisclaimerTooltip.tsx [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
+import VisuallyHiddenDiv from 'components/VisuallyHidden/VisuallyHidden';
 import { useBreakpoint } from 'common/hooks';
 import { tooltipAnchorOnClick } from 'components/InfoTooltip';
+import { v4 as uuidv4 } from 'uuid';
 
 const DisclaimerTooltip: React.FC<StyledTooltipProps> = props => {
   const [isOpen, setIsOpen] = useState(false);
   const isMobile = useBreakpoint(600);
 
+  const idForAccessability = `${uuidv4()}`;
```
Comment (pnavarrc) [inline src/components/InfoTooltip/DisclaimerTooltip.tsx:13]: You don't need the string interpolation, can just use

```ts
const idForAccessability = uuidv4();
```

File: src/components/InfoTooltip/InfoTooltip.tsx [hunk 2]
```diff
@@ -19,6 +20,8 @@ const InfoTooltip: React.FC<StyledTooltipProps> = props => {
     props.trackCloseTooltip();
   };
 
+  const idForAccessability = `${uuidv4()}`;
```
Comment (pnavarrc) [inline src/components/InfoTooltip/InfoTooltip.tsx:23]: Also here

File: src/components/VisuallyHidden/VisuallyHidden.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  overflow: hidden;
+  position: absolute;
+  white-space: nowrap;
+  width: 1px;
+`;
+
+const VisuallyHiddenDiv: React.FC<{
+  content: React.ReactNode;
+  elemId: string;
+}> = ({ content, elemId }) => {
```
Comment (pnavarrc) [inline src/components/VisuallyHidden/VisuallyHidden.tsx:24]: I don't think we need the `VisuallyHiddenDiv` component, we can just pass the id, children and role to the `HiddenDiv` component directly:

```tsx
<HiddenDiv id={idForAccessability} role="tooltip">
  {props.title}
</HiddenDiv>
```

**Review by pnavarrc — APPROVED** [review]
Looks good!

File: src/components/VisuallyHidden/VisuallyHidden.tsx (hunk 3 above)
Comment (chasulin) [reply src/components/VisuallyHidden/VisuallyHidden.tsx:24]: per our chat, keeping it as is

**Pull request merged**